types/lib/sequelize.d.ts

```diff
@@ -673,6 +673,15 @@ export class Sequelize extends Hooks {
   public static afterSync(name: string, fn: (options: SyncOptions) => HookReturn): void;
   public static afterSync(fn: (options: SyncOptions) => HookReturn): void;
 
+  /**
+   * Use CLS with Sequelize.
+   * CLS namespace provided is stored as `Sequelize._cls`
+   * and bluebird Promise is patched to use the namespace, using `cls-bluebird` module.
+   *
+   * @param namespace
+   */
+  public static useCLS(namespace: object): typeof Sequelize;
```

Comment:
Please note that I'm keeping it to `object` unless we want to create an interface with `bind` and `run` defined, I felt like this was too far reaching into cls tho.